test/integration/api/users.js

```diff
@@ -287,6 +287,22 @@ describe('api: /users', () => {
               .expect(401);
           }))));
 
+    it('should return 403 if user doesn\'t have rights to invalidate', testService((service) =>
+      service.login('chelsea', (asChelsea) =>
+        asChelsea.post('/v1/users/reset/initiate?invalidate=true')
+          .send({ email: 'winnifred@getodk.org' })
+          .expect(403)
+          )));
```

Comment:
We generally keep closing parentheses on the same line, so it'd be great to combine these two lines.